prymer/api/picking.py

```diff
@@ -26,10 +26,10 @@
 from pysam import FastaFile
 
 from prymer.api.melting import calculate_long_seq_tm
-from prymer.api.minoptmax import MinOptMax
-from prymer.api.oligo import Oligo
-from prymer.api.primer_pair import PrimerPair
-from prymer.api.span import Span
+from prymer.model import MinOptMax
+from prymer.model import Oligo
+from prymer.model import PrimerPair
+from prymer.model import Span
```

Comment:
Shall we prefer importing from the top-level, here and elsewhere?  Or is there a reason for the preference to import it from model?
```suggestion
from prymer import MinOptMax
from prymer import Oligo
from prymer import PrimerPair
from prymer import Span
```